searchlib/src/main/java/com/yahoo/searchlib/rankingexpression/Reference.java

```diff
@@ -46,6 +49,8 @@ private Reference(String name, Arguments arguments, String output, boolean isIde
         Objects.requireNonNull(name, "name cannot be null");
         Objects.requireNonNull(arguments, "arguments cannot be null");
         this.arguments = arguments;
+        if (featureNames.contains(name))
```

Comment:
I think this should use FeatureNames.isSimpleFeature instead.

Reply:
Requires moving FeatureNames out of config-model, but okay, done.